src/clj/game/cards/resources.clj

```diff
@@ -1536,7 +1536,12 @@
                             (continue-ability state side ab card targets)))}]})
 
 (defcard "Guru Davinder"
-  {:flags {:cannot-pay-net true}
+  {:static-abilities [{:type :cannot-pay-net
+                       :value (constantly true)}
```

Comment:
it has to be a function? why not a literal true?

Reply:
honestly, I don't know why I did that :joy: